src/modules/leaderboardApi.js

```diff
@@ -0,0 +1,53 @@
+const baseURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+
+const createGame = async (name) => {
+  try {
+    const response = await fetch(`${baseURL}games/`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        name,
+      }),
+    });
+
+    const data = await response.json();
+    const gameId = await data.result;
+    return gameId;
+  } catch (err) {
+    return null;
+  }
+};
+
+const submitScore = async (gameId, user, score) => {
+  try {
+    const response = await fetch(`${baseURL}games/${gameId}/scores/`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        user,
```

Comment:
- [ ] After adding the score to the leaderboard if I refresh the page, I can't see the added score on the screen. It's because on every app start(refresh) the new gameId is generated and being used for APIs. Please make sure to fix this issue and after the page refresh, users should be able to see the added scores.
  - I would suggest using hardcoded gameId instead of generating it every time when app loads.